addons/web/static/src/views/no_content_helpers.xml

```diff
@@ -5,6 +5,13 @@
         <div class="o_view_nocontent">
             <div class="o_nocontent_help">
                 <t t-out="noContentHelp"/>
+                <t t-if="showWidgetSampleData and useSampleModel">
```

Comment:
given the changes of the two commits, I'm starting to thing actionHelper should be a Component

@Polymorphe57 
For the remark above and for the changes involving searchModel.facets, can I have a second pair of eyes on this ?

Reply:
> given the changes of the two commits, I'm starting to thing actionHelper should be a Component
> 
> @Polymorphe57 For the remark above and for the changes involving searchModel.facets, can I have a second pair of eyes on this ?

I think you're right. Other remark. This seems incomplete: check all calls to "web.ActionHelper"